apps/maestro/src/config/chains/evm-chains.ts

```diff
@@ -43,6 +43,7 @@ import {
   scroll,
   scrollSepolia,
   sepolia,
+  xrplevmTestnet,
```

Comment:
```suggestion
  xrplEvmTestnet,
```

Reply:
that's the name that exported from `viem/chains` library.